### ISSUE TITLE
I want to hide password in common.runtime.properties. Do I need to use Druid's Password Provider?

### ISSUE THREAD

**Issue by JiHyeongSeo** [apache/druid#9465]
Hello.

I want to hide password in common.runtime.properties. 

According to Druid docs (https://druid.apache.org/docs/0.17.0/operations/password-provider.html#docsNav),

Environment variable password provider provides password by looking at specified environment variable. Use this in order to avoid specifying password in runtime.properties file. e.g

{ "type": "environment", "variable": "METADATA_STORAGE_PASSWORD" }

But I feel that explanation is lacking.

Do I need to use Druid's Password Provider? How can i use Password Provider to hide password in common.runtime.properties?

Thanks.

**Comment by gianm** [comment]
Yes, that's what password providers are for. You can specify that JSON in the `password` property and it will use an environment variable instead.

**Comment by JiHyeongSeo** [comment]
@gianm please would you show me example code ??

**Comment by ankitmani2004** [comment]
Hi. In the runtime properties(common.runtime.properties) file i added below lines:

```json
druid.s3.accessKey={ "type": "environment", "variable": "druid_s3_accessKey" }
```
#druid.s3.secretKey={ "type": "environment", "variable": "druid_s3_secretKey" }
#druid.s3.secretKey=%{druid_s3_secretKey}%
```json
druid.s3.secretKey={ "type": "environment", "variable": "druid_s3_secretKey" }
```

but it did not work. What am I missing here. Thanks.
